docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,241 @@
+# LORIS Developers Guide
+This document covers topics including making PRs, issues, and how to go through a normal LORIS workflow when contributing to the LORIS repository. Many notes in this guide are also relevant to contributing code on a fork or instance of LORIS.
+
+## Links to know, read, and follow
+* All markdowns in the Loris GitHub under [docs/](../..) 
+* [Contributing.md](../../../CONTRIBUTING.md) - How to contribute to Loris
+* [Coding Standards](../../CodingStandards.md)
+* [Pull Request Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) and [Code Review guidelines](Code_Review_Guide.md)
+
+## 1. Working off your fork
+During installation, you made your own fork of LORIS. Now you can add two remote options in order to switch between your fork and “aces”.
+
+    git remote rename origin aces
+    git remote add my_fork <fork_url>
+             
+Your fork url will usually have the format:  
+    `https://github.com/github-username/LORIS`  
+    
+Any development should be done on your fork of LORIS, and **not** on aces/LORIS. 
+Before starting any PR, be mindful of which branch you want to base off of (main vs release-specific branch).   
+
+You will make a different branch for each PR or change that you are making.  
```

Comment:
```suggestion
You will make a different branch for each PR or change that you are making.  
In general, smaller pull requests / changes are easier to work with in our team review context and will flow more smoothly through the review process.  
```

Reply:
This information was added in the internal document but I will add it here as well. Thanks 👍

Reply:
This thread looks like it should be marked as resolved? (The paragraph is already included below..)